lib/src/password-input/types.ts

```diff
@@ -31,19 +31,22 @@ type Props = {
    * This function will be called when the user types within the input
    * element of the component. An object including the current value and
    * the error (if the value entered is not valid) will be passed to this
-   * function. If there is no error, error will be null.
+   * function. If there is no error, error will not be defined.
    * */
-  onChange?: (val: { value: string; error: string | null }) => void;
+  onChange?: (val: { value: string; error?: string }) => void;
   /**
    * This function will be called when the input element loses the focus.
    * An object including the input value and the error (if the value entered is
-   * not valid) will be passed to this function. If there is no error, error will be null.
+   * not valid) will be passed to this function. If there is no error, error will not be defined.
    */
-  onBlur?: (val: { value: string; error: string | null }) => void;
+  onBlur?: (val: { value: string; error?: string }) => void;
   /**
-   * If it is defined, the component will change its appearance, showing
-   * the error below the password input component. If it is not defined, the
-   * error messages will be managed internally, but never displayed on its own.
+   * If it is a defined value and also a truthy string, the component will
+   * change its appearance, showing the error below the password input
+   * component. If the defined value is an empty string, it will reserve a
+   * space below the component for a future error, but it would not change
+   * its look. In case of being undefined or null, both the appearance and
+   * the space for the error message would not be modified.
    */
   error?: string;
```

Comment:
Remember to also update here `pattern`, `minLength` and `maxLength` prop type descriptions.